pkg/blockbuilder/metrics.go

```diff
@@ -37,8 +28,9 @@ func newBlockBuilderMetrics(reg prometheus.Registerer) blockBuilderMetrics {
 	}, []string{"success"})
 
 	m.processPartitionDuration = promauto.With(reg).NewHistogramVec(prometheus.HistogramOpts{
-		Name:                        "cortex_blockbuilder_process_partition_duration_seconds",
-		Help:                        "Time spent processing one partition.",
+		Name: "cortex_blockbuilder_process_partition_duration_seconds",
```

Comment:
I think this one is now redundant with the similar `consume_job` metric. It's in some dashboards, though.